### PR TITLE
[DOCS] Use named snippets part 8 

### PR DESCRIPTION
**Pull request by Shinnnyshinshin**
Changes proposed in this pull request:

Use named snippets instead of line numbers in code block references.

### CODE REVIEW

File: tests/integration/docusaurus/connecting_to_your_data/filesystem/pandas_yaml_example.py [hunk 1]
```diff
@@ -25,15 +29,21 @@
...
 # In normal usage you'd set your path directly in the yaml above.
 datasource_yaml = datasource_yaml.replace("<PATH_TO_YOUR_DATA_HERE>", "../data/")
 
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/filesystem/pandas_yaml_example.py test_yaml_config">
 context.test_yaml_config(datasource_yaml)
+# </snippet>
 
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/filesystem/pandas_yaml_example.py add_datasource">
 context.add_datasource(**yaml.load(datasource_yaml))
+# </snippeto>
```
Comment (anthonyburdi) [inline tests/integration/docusaurus/connecting_to_your_data/filesystem/pandas_yaml_example.py]:
```suggestion
# </snippet>
```

File: docs/guides/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe.md [hunk 2]
```diff
@@ -39,12 +39,12 @@ What used to be called a “Batch” in the old API was replaced with <Technical
...
      First, import these necessary packages and modules.
 
-     ```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_dataframe.py#L2-L11
+     ```python name="tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_dataframe.py imports"
      ```
 
      Load an on-disk <TechnicalTag tag="data_context" text="Data Context" /> (ie. from a `great_expectations.yml` configuration) via the `get_context()` command:
 
-     ```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_dataframe.py#L14
+     ```python # <snippet name="tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_dataframe.py get_context"
```
Comment (anthonyburdi) [inline docs/guides/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe.md]: Not showing up for some reason: 
image

File: docs/guides/connecting_to_your_data/in_memory/spark.md [hunk 3]
```diff
@@ -88,14 +87,12 @@ Save the configuration into your `DataContext` by using the `add_datasource()` f
...
-
-```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/in_memory/spark_yaml_example.py#L46
+```python name="tests/integration/docusaurus/connecting_to_your_data/in_memory/spark_yaml_example.py add datasource"
 ```
 
 </TabItem>
 <TabItem value="python">
-
-```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/in_memory/spark_python_example.py#L46
+```python name="tests/integration/docusaurus/connecting_to_your_data/in_memory/spark_python_example.py add datasource"
```
Comment (anthonyburdi) [inline docs/guides/connecting_to_your_data/in_memory/spark.md]: For some reason these are not rendering: 
image

**Review by anthonyburdi — CHANGES_REQUESTED** [review]
Just a couple tweaks needed

File: docs/guides/connecting_to_your_data/in_memory/spark.md (hunk 3 above)
Comment (Shinnnyshinshin) [reply docs/guides/connecting_to_your_data/in_memory/spark.md]: I think there was some tab-related issue with this doc. Will check it out

**Review by anthonyburdi — APPROVED** [review]
LGTM!

**Pull request merged**